packages/nitro-protocol/package.json

```diff
@@ -32,7 +32,6 @@
   "homepage": "https://github.com/statechannels/nitro-protocol/",
   "scripts": {
     "build": "run-s clearContracts clearLib contract:compile build:typescript",
-    "build:ci": "yarn build",
```

Comment:
As a note, the reason why I think this can be skipped is that `wallet` uses `tsc -b` which uses TypeScript references to auto-run `tsc -b` in the `nitro-protocol` package.

Reply:
All packages that are dependents need to be built in the `prepare` step, so `nitro-protocol` isn't out of date once circle is running the `build` job. Is the `-b` needed considering this?